environment_meson.yml

```diff
@@ -45,5 +45,6 @@ dependencies:
   # For CLI
   - rich-click
   - click
+  - doit==0.36.0
   - pip:
-      - doit==0.35.0
+      - pydevtool==0.2.0
```

Comment:
Do you plan to put it on Conda-forge (I can help if wanted).

Reply:
ok sure no problem. I will ping you shortly there 👍

Reply:
PR is here https://github.com/conda-forge/staged-recipes/pull/18753

Reply:
Now available on Conda-forge. Can you update and remove the pip section? And while doing it can you also update the other env file? We don't need pip now.

Reply:
I thought as well but the whole cli deps were added here too. Anyway, these files will be merged soon enough.